server-side/controllers/message.controller.js

```diff
@@ -0,0 +1,94 @@
+
+
+import Message from '../models/Message.model.js';
+import MessageType from '../models/MessageType.model.js';
+
+
+
+export const getMessages = async (req, res) => {
+  try {
+    const messages = await Message.find().select('-__v');
```

Comment:
do the following for all controller files:
1. access to collections and logics should be in service file.
2. add return statements for all responses (res.send, res.send, next etc) and status code
3. use restructuring objects, for example: const {userId} = req.params insteadof const userId= req.params.userId